scompose/project/project.py

```diff
@@ -347,7 +366,11 @@ def down(self, names=None):
            names: a list of names of instances to bring down. If not specified, we
            bring down all instances.
         """
-        names = names or self.get_instance_names()
+        if not names:
+          names = list(self.get_instance_names())
```

Comment:
This might be my eyes, but I don't think this is enough spaces here? (should be 4).

Reply:
Also, get_instance_names should return a list so we don't need to do that here. I think I commented on this above when there was a reference to keys() without list wrapping it.